spk/emby/src/dsm-control.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+
+# Package
+PACKAGE="emby"
+DNAME="Emby"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+USER="${PACKAGE}"
+
+MONO_PATH="/usr/local/mono/bin"
+MONO="${MONO_PATH}/mono"
+EXE_FILE="${INSTALL_DIR}/share/emby/MediaBrowser.Server.Mono.exe"
+
+EXTRA_LIBS="/usr/local/mediainfo/lib:/usr/local/imagemagick/lib:${INSTALL_DIR}/lib"
+FFMPEG="/usr/local/ffmpeg/bin"
+
+DATA_DIR="${INSTALL_DIR}/var"
```

Comment:
Personally, I'm not a fan of adding this type of variable, where it's not much of a shortcut at all. Simply referring to `{INSTALL_DIR}/var` where you need it works fine. (PID_FILE and `-programdata` in COMMAND)